storm-multilang/dotnet/src/main/dotnet/Dotnet.Storm.Adapter/Test/TestAPI.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+using CommandLine;
+using Dotnet.Storm.Adapter.Channels;
+using Dotnet.Storm.Adapter.Components;
+using Dotnet.Storm.Adapter.Logging;
+using Dotnet.Storm.Adapter.Messaging;
+using Dotnet.Storm.Adapter.Serializers;
+using log4net;
+using log4net.Config;
+using log4net.Core;
+
+namespace Dotnet.Storm.Adapter.Test
+{
+    public static class TestAPI
+    {
+        public static Component CreateComponent(Type type)
+        {
+            Channel.Instance = new CacheChannel()
+            {
+                Serializer = new JsonSerializer()
+            };
+
+            Component comp = (Component)Activator.CreateInstance(type);
+
+            Component.Context = new StormContext();
```

Comment:
Consider adding ability to pass in parameters so that we can pass Context and Configuration properties.
Such as Context.ComponentId, Configuration["azure.spout.emit.frequency"] and Configuration["azure.service.name"]

Reply:
This is a tricky one. This is because contexts of components are extracted from messages received by each component instead of read from file.